### PR TITLE
[FIX] #4887 Don't break on po files containing just hashes

### PR DESCRIPTION
**Pull request by ccomb**
Obvious fix for #4887

### CODE REVIEW

File: openerp/tools/translate.py [hunk 1]
```diff
@@ -329,6 +329,8 @@ def next(self):
                 elif line.startswith('#,') and (line[2:].strip() == 'fuzzy'):
                     fuzzy = True
                 line = self.lines.pop(0).strip()
+            if 0 == len(self.lines):
```
Comment (pedrobaeza) [inline openerp/tools/translate.py]: Better `if not len(self.lines): ...`

Comment (xmo-odoo) [inline openerp/tools/translate.py]: `len` shouldn't even be necessary, since `self.lines` is a list.

Comment (pedrobaeza) [inline openerp/tools/translate.py]: Yeah, right, fast-review error...

Comment (ccomb) [inline openerp/tools/translate.py]: I wrote that to be consistent with lines 305 and 339.  Should I change these other lines too?

Comment (pedrobaeza) [inline openerp/tools/translate.py]: Well, I don't think so, because this is a better Python construction, but let's see what @xmo-odoo

Comment (xmo-odoo) [inline openerp/tools/translate.py]: The recommendation is generally to write better code when you change it (if you can), but not mix new fixes with refactoring/rewriting of working code.

**Comment by yvaucher** [comment]
:+1:

**Comment by mart-e** [comment]
merged at 737728a7a05cf849ced6e53d6d776e10b67dc804, thanks

Also, no need to [create an issue](https://github.com/odoo/odoo/wiki/Contributing#reporting-issues) just for the report is you have already a fix. Just create a pull request with the description of the issue.

**Comment by pedrobaeza** [comment]
Thank you very much!

**Comment by ccomb** [comment]
thx for the merge, ok I don't create an issue for the latest PRs

**Comment by pedrobaeza** [comment]
@mart-e One question: is this going to be applied also on 8.0 and master branches? The behaviour is also the same.

**Comment by mart-e** [comment]
@pedrobaeza yes, we periodically port every patch to upper version (that's the reason you should always make your pr [against the lowest version](https://github.com/odoo/odoo/wiki/Contributing#against-which-version-should-i-submit-a-patch)).
The only time it's worth to make more than one pull request is when the fix is different (e.g. workaround in stable, clean solution by adding a new field in master).

**Comment by pedrobaeza** [comment]
Thanks for the clarification.

It's for cancelling corresponding OCB PRs.

**Pull request closed without merging**